app/controllers/api/v3/scenarios_controller.rb

```diff
@@ -88,7 +88,7 @@ def create
 
         @scenario = Scenario.new
 
-        if scaler_attributes && ! attrs[:descale]
+        if scaler_attributes && ! attrs[:descale] && ! Area.derived?(attrs.fetch(:area_code))
```

Comment:
> Should it be possible to descale a scenario from a derived dataset? What would we expect? To get a scenario with the base_dataset and the inputs scaled up to the size of Holland? 🤔

If you were to scale a scenario which uses a derived dataset (e.g. to 10 households), then descaling would return it back to the size of the *derived* region (Ameland), not the original NL dataset.

It should not be possible (at this stage) to take a scenario with a derived dataset and convert it to a full-size NL scenario.

As far as ETEngine is concerned, I think it should see the derived dataset and the original dataset as two entirely different things: they're just two different datasets; how Atlas+Refinery created them is not important.